Dockerfile

```
@@ -7,6 +7,6 @@ COPY src/ src/
 RUN shards build --release --production --no-debug && strip bin/*
 
 FROM alpine:latest
-RUN apk add --no-cache libgcc libevent pcre libssl1.1 nftables
+RUN apk add --no-cache libgcc libevent pcre2 libssl3 nftables
```

```suggestion
RUN apk add --no-cache libgcc libevent nftables
```

Shouldn't we be able to ignore this then, as they are included in the base alpine image? Or is it best practice to explicitly include dependencies here?

---

Or aha, we aren't using our own crystal image here, but the apline base. NVM then

---

This runs on a base, base image (see `alpine:latest` directive). The builder has the dev pkgs for pcre2 and libssl3, but not the "runtime image".

`sparoid` depends on pcre2, hence it's needed here. I am honestly not sure about `libssl3`, it looks like it's already installed in the image so it's a no-op anyway, better safe than sorry? :-)

---

Yeah, noticed. Looked at row # 1, that FROM image and not Line # 9 😄 

All good!